logger_formatter.go

```diff
@@ -1,6 +1,6 @@
 package buffalo
 
-// I really don't want to have this, but until (if) https://github.com/sirupsen/logrus/pull/606 is merged we're stuck with all this code. And yes, this is ALL needed just to remove some blank space in the logs
+// I really don't want to have this, but until (if) https://github.com/Sirupsen/logrus/pull/606 is merged we're stuck with all this code. And yes, this is ALL needed just to remove some blank space in the logs
```

Comment:
The repo is now lower case `sirupsen`. Can you please undo these import changes?